src/main/java/net/sf/jabref/gui/exporter/SaveDatabaseAction.java

```diff
@@ -85,6 +91,8 @@ public void init() throws Throwable {
 
             panel.frame().output(Localization.lang("Saving database") + "...");
             panel.setSaving(true);
+        } else if (filePath.isPresent()) {
+            saveAs(filePath.get().toFile());
```

Comment:
(i) This is just going the same way as the save as functionality
(ii) I wanted to keep that general. It's fully independent of the database location. If a target file is already given, it should only save it there without asking.